omero/sysadmins/windowsmigration.txt

```diff
@@ -0,0 +1,45 @@
+Migration from OMERO 5.2 on Windows to OMERO 5.3 on Linux
+=========================================================
+
+Backup (Windows)
+----------------
+
+- Create a dump of the OMERO SQL database, either using
+  `pgAdmin <https://www.pgadmin.org/>`_ or the :command:`pg_dump` command line
+  program.
+- Backup the OMERO data directory (e.g. :file:`C:\\OMERO`).
```

Comment:
The point to get across is that they *must* maintain a pristine copy of the original system if they want us to help them find errors later. *If* they are going to just leave the Windows data where it is (and for example rsync it over), then at this point they don't need to do anything. But if they are using NFS or something, then at this point they should make another copy for later.